spec/amber/dsl/server_spec.cr

```diff
@@ -124,6 +124,45 @@ module Amber
           end
         end
       end
+
+      describe "routes" do
+        context "with a namespace" do
+          server = Amber::Server.instance
+          server.handler.pipeline.clear
+
+          Amber::Server.configure do
+            pipeline :web do
+            end
+
+            routes :web do
```

Comment:
Can you add one more test?  You can add a namespace to a `routes` as follows:
```
routes :api, "/api" do
  namespace "/test" do
    resources "/user", UserController
  end
end
```
Can you verify this creates `/api/test/user`?

Reply:
Done :+1: